src/pretix/control/views/item.py

```diff
@@ -1326,7 +1326,8 @@ def plugin_forms(self):
             if isinstance(resp, (list, tuple)):
                 forms.extend(resp)
             else:
-                forms.append(resp)
+                if resp:
+                    forms.append(resp)
```

Comment:
See above. Exit early, then decide what to do?
```suggestion
                forms.append(resp)
```